windows_containers/windows-containers-release-notes-10-15-x.adoc

```diff
@@ -33,6 +33,19 @@ endif::openshift-origin[]
 
 Starting with this release, y-stream releases of the WMCO will be in step with {product-title}, with only z-stream releases between {product-title} releases. The WMCO numbering reflects the associated {product-title} version in the y-stream position. For example, the current release of WMCO is associated with {product-title} version 4.15. Thus, the numbering is WMCO 10.15.z.
 
+[id="wmco-10-15-2"]
+== Release notes for Red{nbsp}Hat Windows Machine Config Operator 10.15.2
+
+This release of the WMCO provides new features and bug fixes for running Windows compute nodes in an {product-title} cluster. The components of the WMCO 10.15.2 were released in link:https://access.redhat.com/errata/RHBA-2024:TBD[RHBA-2024:TBD].
+
+[id="wmco-10-15-2-bug-fixes"]
+=== Bug fixes
+
+* Previously, on Azure clusters the WMCO would check if an external cloud controller (CCM) was being used on the cluster. CCM use is the default. If a CCM is being used, the Operator would adjust configuration logic accordingly. Because the status condition that the WMCO used to check for the CCM has been removed, the WMCO acted as if a CCM was not in use. This fix removes the check. As a result, the WMCO always configures the required logic on Azure clusters.
+
+* Previously, the kubelet was unable to authenticate with private ECR registries. Because of this error, the kubelet was not able to pull images from these registries. With this fix, the kubelet is able to pull images from these registries as expected. (link:https://issues.redhat.com/browse/OCPBUGS-26602[*OCPBUGS-26602*])
```

Comment:
```suggestion
* Previously, the kubelet was unable to authenticate with private Elastic Container Registries (ECR). Because of this error, the kubelet was not able to pull images from these registries. With this fix, the kubelet is able to pull images from these registries as expected. (link:https://issues.redhat.com/browse/OCPBUGS-26602[*OCPBUGS-26602*])
```